Podfile

```diff
@@ -2,16 +2,19 @@ platform :ios, '8.0'
 source 'https://github.com/CocoaPods/Specs.git'
 
 target 'Signal' do
-    pod 'ATAppUpdater'
+	pod '25519', :inhibit_warnings => true
```

Comment:
nits:
- Weird indent. Can you make sure your editor is not configured to insert tabs?
- elsewhere we're using the `key: 'value'` hash syntax, not the `'key' => 'value'` hash syntax.

not just nits:

- What version of cocoapods are you using?  I'm using the bundler specified version and unable to `pod install`

with error:
```
NoMethodError - undefined method `match' for 25519:Fixnum
/Users/mkirk/.rbenv/versions/2.0.0-p648/lib/ruby/gems/2.0.0/gems/cocoapods-core-1.3.1/lib/cocoapods-core/dependency.rb:348:in `from_string'
```
probably due to this not being merged: https://github.com/WhisperSystems/Curve25519Kit/pull/1/commits/c87d1eb0ad338389add67ae1f7e0b85582986187

If you're still OK going forward with the framework friendly stuff, I can rebase the relevant PRs. The previous motivation for it is no longer immediately pressing, but annoyances like this keep popping up.

Also, we probably shouldn't be inhibiting warnings for this library anyway, which is code that we're maintaining.

Reply:
> Weird indent. 

Fixed.

> elsewhere we're using the key: 'value' hash syntax, not the 'key' => 'value' hash syntax.

That doesn't work for me.  Is that syntax only supported in podspecs?  I'm following the examples in the cocoapods docs, IIRC

Reply:
> What version of cocoapods are you using? I'm using the bundler specified version and unable to pod install

Fixed, I removed `25519` altogether from this podfile since as you point out we maintain the code and don't want to suppress its warnings.